src/fides/api/ops/task/graph_task.py

```diff
@@ -105,7 +102,11 @@ def result(*args: Any, **kwargs: Any) -> Any:
                         f"{self.traversal_node.address.value}", 0
                     )  # Cache that the erasure was performed in case we need to restart
                     return 0
-                except (CollectionDisabled, NotSupportedForCollection) as exc:
+                except (
+                    CollectionDisabled,
+                    NotSupportedForCollection,
+                    SkippingConsentPropagation,
+                ) as exc:
```

Comment:
This is in the wrapper that wraps nodes in the graph that also creates the execution logs - here is where raising a SkippingConsentPropagation in execution will create a skipped log for that node.